src/main/resources/story/web/login.story

```diff
@@ -0,0 +1,33 @@
+Scenario: Log-in with invalid data
+Given I am on the main application page
+When I click on element located by `caseSensitiveText(Log in)`
+When I wait until element located `id(user)` appears
+When I enter `#{generate(Internet.emailAddress)}` in field located `id(user)`
+When I click on element located `id(login)`
+When I wait until element located `id(password)` appears
+When I enter `#{generate(Internet.password)}` in field located `id(password)`
+When I click on element located `id(login)`
+Then number of elements found by `By.xpath(//*[@id="error" and @class="quick-switch"])` is = `1`
+
+Scenario: Log-in
+Given I am on the main application page
+When I click on element located by `caseSensitiveText(Log in)`
+When I wait until element located `id(user)` appears
+When I enter `${email}` in field located `id(user)`
+When I click on element located `id(login)`
+When I wait until element located `id(password)` appears
+When I enter `${password}` in field located `id(password)`
+When I click on element located `id(login-submit)`
+
+Scenario: Log-out
+When I wait until element located `By.xpath(//*[@data-testid="header-member-menu-button"])` appears
+When I click on element located `By.xpath(//*[@data-testid="header-member-menu-button"])`
+When I wait until element located `By.xpath(//*[@data-testid="header-member-menu-logout"])` appears
+When I click on element located `By.xpath(//*[@data-testid="header-member-menu-logout"])`
+When I wait until element located `id(logout-submit)` appears
+When I click on element located `id(logout-submit)`
+When I wait until element located `By.xpath(//*[@href="/login"])` appears
+
+Scenario: Composite step
```

Comment:
Remove this scenario
Composite step is not a functionality we'd like to test here^ so there's no need for it in a seperate scenario